src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/callbacks/ioc/PaymentsService.java

```diff
@@ -43,7 +44,8 @@ public PaymentsService(
         @Value("${payments.api.service}") String service,
         @Value("${payments.api.siteId}") String siteId,
         @Value("${payments.api.currency}") String currency,
-        @Value("${payments.api.description}") String description
+        @Value("${payments.api.description}") String description,
+        @Value("${feature_toggles.auto_cancel_payments}") boolean autoCancel
```

Comment:
Probably not for this PR but having a compose object for all the configuration properties of feature_toggles would be more cleaner. And we can inject the object `FeatureToggles` i.e

```suggestion
        FeatureToggles  featureToggles
```

```suggestion

@Validated
@Component
@ConfigurationProperties(prefix = "feature_toggles")
public class FeatureToggles { 
     @NotEmpty
     private String autoCancelPayments;
     ...//other feature toggles here 
        }
```

Reply:
Nice idea, wanna raise a tech-debt for that @sabahirfan ?

Reply:
yes please. 👍